src/cdk/table/table.ts

```diff
@@ -455,6 +457,13 @@ export class CdkTable<T> implements AfterContentChecked, CollectionViewer, OnDes
   }
   private _fixedLayout: boolean = false;
 
+  /**
+   * Emits when the table completes rendering a set of data rows based on the latest data from the
+   * data source, even if the set of rows is empty.
+   */
+  @Output()
+  readonly dataRowsChange = new EventEmitter<void>();
```

Comment:
I'm not 100% sure on the naming. Wouldn't it also fire if the rows remained the same, but some of cells changed?

Reply:
Good point, it's not quite representing that the rows changed, just that we attempted a render based on a data change. I think the placement of the `emit` is what clients want (they just want to know the table reacted to the data). Any suggestions on the name?

Reply:
What about something like `dataChanged` or `contentChanged`?

Reply:
Renamed to `contentChanged`, SGTM